lib/graphql/schema.ts

```diff
@@ -1355,13 +1332,44 @@ export type Invoice = {
 
 export type InvoiceProductOutput = {
   __typename?: 'InvoiceProductOutput';
-  id: Scalars['String'];
+  id?: Maybe<Scalars['String']>;
```

Comment:
Why is the `id` now optional for `InvoiceProductOutput `, it should be mandatory. The product id should always be in the BE response. I know it is not part of your pr, we may have missed it during @kolyose 's pr review.

Reply:
In `services/backendService/src/core/graphql/types/invoicing/product.ts` I see
```
export class UserProductInput {
  @Field({ nullable: true })
  id?: string;
  ...
  ```
  
  I think it was done in order to be able to use the same code for input and output. And for input id is optional

Reply:
I saw it too, but then we're creating two types based on the `InvoiceProduct` which itself extends `UserProductInput`
Those two types are exactly the same, so if we decide that the id is optional for both then we can just merge them into one.  But then I remembered that Sergey had types issues, and that's why we had to duplicate them, but I can find the conversation about it in ou slack channel 😅 
```
@InputType()
export class InvoiceProductInput extends InvoiceProduct {}

@ObjectType()
export class InvoiceProductOutput extends InvoiceProduct {}
```

Reply:
It can be fixed later, as discussed here: https://kontist.slack.com/archives/G01J0BJAA3U/p1618321820029700